### ISSUE TITLE
`npm install` fails

### ISSUE THREAD

**Issue by romainmenke** [cdoublev/css#1]
https://github.com/cdoublev/css/blob/4f882bfff889a8b4e67d37ed2bb271369c67bf9e/package.json#L36

```
pm ERR! command failed
npm ERR! command sh -c npm run build:interface && npm run build:definitions
npm ERR! > @cdoublev/css@0.1.0 build:interface
npm ERR! > node ./scripts/interface.js
npm ERR! 
npm ERR! 
npm ERR! > @cdoublev/css@0.1.0 build:definitions
npm ERR! > npm run build:definitions:extract && npm run build:definitions:initial
npm ERR! 
npm ERR! 
npm ERR! > @cdoublev/css@0.1.0 build:definitions:extract
npm ERR! > node ./scripts/extract.js
npm ERR! 
npm ERR! Error: Unhandled duplicate definitions of <fit-content()>
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:712:27
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at addTypes (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:676:17)
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:761:9
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at addProperties (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:732:17)
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:852:9
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at build (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:848:36)
npm ERR! Please report this issue: https://github.com/cdoublev/css/issues/new
npm ERR! node:internal/process/promises:289
npm ERR!             triggerUncaughtException(err, true /* fromPromise */);
npm ERR!             ^
npm ERR! 
npm ERR! Error: Unhandled duplicate definitions of <fit-content()>
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:712:27
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at addTypes (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:676:17)
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:761:9
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at addProperties (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:732:17)
npm ERR!     at /Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:852:9
npm ERR!     at Array.forEach (<anonymous>)
npm ERR!     at build (/Users/romainmenke/.npm/_cacache/tmp/git-clonesYPidqwseic3/scripts/extract.js:848:36)
npm ERR! 
npm ERR! Node.js v21.7.1
```

I know this isn't published yet and probably not yet expected to be used, but I wanted to play around with it and unfortunately installing fails.

I suspect that it is caused by the `prepare` script.

Which seems to run when adding it as a dependency like this : `"@cdoublev/css": "git+https://github.com/cdoublev/css.git"`

I thought I could work around this by doing `npm install --ignore-scripts` but that didn't seem to have any effect.

--------

Feel free to ignore and close this issue if fixing it isn't worth it for you at this time :)

**Comment by cdoublev** [comment]
Yeah, you are right on every point.

I stopped rigorously versioning and maintaining this repository a long time ago. It slowed me down too much.

Ideally this error should be fixed in the spec (CSS Grid). It can be fixed by appending `<fit-content()>` and `<minmax()>` into `excluded.types` in [`scripts/extract.js`](https://github.com/cdoublev/css/blob/4f882bfff889a8b4e67d37ed2bb271369c67bf9e/scripts/extract.js#L435). But other uncomitted changes are missing.

I am going to move this to the top of my to-do list. I procrastinate too much on an important implementation decision but I need to move forward and it might be helpful to have people like you to play around.

**Comment by cdoublev** [comment]
`npm i cdoublev/css` should work now. I updated the README with details about the expected global object or the argument of `install()`, which should be `window` or a mock. It was unclear.

I do not know if you wanted to play with CSSOM, the [CSS parser](https://github.com/cdoublev/css/blob/00196f793e2bc33fae7cde7e548be622bca2e6cc/lib/parse/parser.js#L862), run tests, or something. Expect to see some tests fail (about 10 of 1000) because I did not committed all my local changes, for various reasons. Anyway, happy to answer your questions and fix any issues.
